src/test/java/blackjack/domain/result/JudgeTest.java

```diff
@@ -0,0 +1,113 @@
+        Players players = new Players(List.of(zero, corinne), dealer);
+        GameResult result = Judge.calculateGameResult(players);
+
+        assertAll(() -> {
+            assertThat(result.getDealerResult().getWin().getCount()).isEqualTo(2);
+            assertThat(result.getDealerResult().getLose().getCount()).isEqualTo(0);
+            for (ParticipantResult participantResult : result.getParticipantResults()) {
+                assertThat(participantResult.getResult()).isEqualTo(Result.LOSE);
+            }
+        });
+    }
+
+    @Test
+    @DisplayName("참가자가 Burst고 딜러는 Burst가 아니라면 딜러가 승리한다.")
+    void dealerWinWhenNotBurstAndParticipantBurst() {
+        Participant zero = new Participant(
+                List.of(new Card(Type.SPADE, Score.KING), new Card(Type.HEART, Score.SIX)), "zero");
+        zero.addCard(new Card(Type.CLOVER, Score.TEN));
+
+        Participant corinne = new Participant(
+                List.of(new Card(Type.CLOVER, Score.SIX), new Card(Type.HEART, Score.TEN)), "corinne");
+        corinne.addCard(new Card(Type.DIAMOND, Score.SIX));
+
+        Dealer dealer = new Dealer(List.of(new Card(Type.HEART, Score.ACE), new Card(Type.DIAMOND, Score.KING)));
+        Players players = new Players(List.of(zero, corinne), dealer);
+        GameResult result = Judge.calculateGameResult(players);
+
+        assertAll(() -> {
+            assertThat(result.getDealerResult().getWin().getCount()).isEqualTo(2);
+            assertThat(result.getDealerResult().getLose().getCount()).isEqualTo(0);
+            for (ParticipantResult participantResult : result.getParticipantResults()) {
+                assertThat(participantResult.getResult()).isEqualTo(Result.LOSE);
+            }
+        });
+    }
+
+    @Test
+    @DisplayName("참가자와 딜러 모두 Burst 라면 딜러가 승리한다.")
+    void dealerWinWhenBurstAndParticipantBurst() {
+        Participant zero = new Participant(
+                List.of(new Card(Type.SPADE, Score.KING), new Card(Type.HEART, Score.SIX)), "zero");
+        zero.addCard(new Card(Type.CLOVER, Score.TEN));
+
+        Participant corinne = new Participant(
+                List.of(new Card(Type.CLOVER, Score.SIX), new Card(Type.HEART, Score.TEN)), "corinne");
+        corinne.addCard(new Card(Type.DIAMOND, Score.SIX));
+
+        Dealer dealer = new Dealer(List.of(new Card(Type.HEART, Score.ACE), new Card(Type.DIAMOND, Score.KING)));
+        dealer.addCard(new Card(Type.CLOVER, Score.ACE));
+
+        Players players = new Players(List.of(zero, corinne), dealer);
+        GameResult result = Judge.calculateGameResult(players);
+
+        assertAll(() -> {
+            assertThat(result.getDealerResult().getWin().getCount()).isEqualTo(2);
+            assertThat(result.getDealerResult().getLose().getCount()).isEqualTo(0);
+            for (ParticipantResult participantResult : result.getParticipantResults()) {
+                assertThat(participantResult.getResult()).isEqualTo(Result.LOSE);
+            }
+        });
```

Comment:
```suggestion
        assertAll(
            () -> assertThat(result.getDealerResult().getWin().getCount()).isEqualTo(2),
            () -> assertThat(result.getDealerResult().getLose().getCount()).isEqualTo(0),
            () -> assertThat(result.getParticipantResults().get(0).getResult()).isEqualTo(Result.LOSE);
        );
```
형태로 많이 쓰는거 같은데요.
assertAll을 쓰는 이유는 단순히 assertThat을 나열했을 때 첫번째 assert구문에서 성공하지 못할 경우 다음 assert구문을 실행하지 않는데 assertAll은 처음에 실패하더라도 다음 구문들을 실행하기 때문인데요.
혹시 코린처럼 작성한 경우에도 처음에 실패하더라도 다음 구문이 실행되나요??